packages/overlay/src/integrations/sentry/tabs/TracesTab.tsx

```diff
@@ -142,42 +140,40 @@ export function TraceSplitViewLayout({ trace, span, aiConfig }: TraceSplitViewLa
 }
 
 export default function TracesTab() {
-  const context = useSpotlightContext();
-  const { allTraces, localTraces } = useSentryTraces();
-  const [showAll, setShowAll] = useState(() => !context.experiments["sentry:focus-local-events"]);
-  const onShowAll = useCallback(() => setShowAll(true), []);
-  const visibleTraces = showAll ? allTraces : localTraces;
-  const [searchQuery, setSearchQuery] = useState("");
-  const [activeFilters, setActiveFilters] = useState<string[]>([]);
-  const { TRACE_FILTER_CONFIGS, filteredTraces } = useTraceFiltering(visibleTraces, activeFilters, searchQuery);
-  const hiddenItemCount = allTraces.length - visibleTraces.length;
-
-  const traceData = {
-    filtered: filteredTraces,
-    all: allTraces,
-    visible: visibleTraces,
-    hiddenItemCount: hiddenItemCount,
+  const traceData = useTraceData();
+
+  const traceDataForList: TraceListData = {
```

Comment:
These variable names are gonna kill me 🤣  How about:
```suggestion
  const traces: TraceDataContents = {
```